src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -956,4 +956,10 @@
   <data name="Run_NotAccessibleWarning.Text" xml:space="preserve">
     <value>Please define an activation phrase or allow this plugin for the global results to use it</value>
   </data>
+  <data name="Run_AllPluginsDisabled.Text" xml:space="preserve">
+    <value>PowerToys Run is not usable. Enable at least one plugin to use it.</value>

```

Comment:
I think your warning message sounds like a technical issue and I suggest to ask the user to enable a plugin.

So I suggest this text which imo better describes the problem and is a bit more friendly:
```suggestion
    <value>PowerToys Run can't provide any results without plugins. Please enable at least one plugin.</value>
```

Reply:
@enricogior
Should we wtite the message with or without dots? The other (per plugin) messages have no dot at the end?

Reply:
In the Windows 10 Settings, the label for controls and the section names don't have the dot, but descriptions and warnings do:
![image](https://user-images.githubusercontent.com/3206696/108813620-ae5e2a00-75b1-11eb-9d63-9e5f2905e183.png)

Reply:
Then ee should give all warning a dot.

cc: @mykhailopylyp

Reply:
@htcfreek @mykhailopylyp  I'd add the . in the config file of each plugin in the description itself, and not append it in Settings.

Reply:
@niels9001 Can you correct this for the other warnings and descriptions in settings?